docs/cloud-native-security/cloud-workload-protection.asciidoc

```diff
@@ -0,0 +1,14 @@
+[[cloud-workload-protection]]
+= Cloud workload protection
+
+Cloud workload protection helps you monitor and understand what's happening on your Linux hosts. It uses the Elastic Defend integration to capture Linux process data, including commands executed and output generated during shell sessions.
```

Comment:
I took a crack at this sentence:

```
Cloud workload protection helps you monitor and protect your Linux hosts and Kubernetes runtimes. It uses the Elastic Defend integration to capture cloud workload telemetry containing process, file and network activity. In Kubernetes environments, the agent further enriches this telemetry with metadata allowing practitioners to isolate events in their cloud topography. 

This telemetry allows for the automation of the identification of cloud threats with out-of-the-box detection rules and machine learning models. The system can then generate alerts based on these detections, which the user can follow with a remediation or response. 
```

I actually think we should come back to the overall description once the use cases are spelled out. I would recommend @nick-alayil as the person to iterate these use cases.